lib/src/components/NxTableCell/NxTableCell.tsx

```diff
@@ -58,6 +71,30 @@ const NxTableCell = function NxTableCell(props: Props) {
     maskedSort = <NxFontAwesomeIcon icon={faSort} />;
   }
 
+  const filterContent = onFilterChange && (
+    <NxFilterInput disabled={isFilterDisable}
```

Comment:
I'm not sure we want to build the construction of the filters into the table cell component.  I think the table components might just need to set up the styles for the filter row overall (i.e. the white background) and then it would be up to the caller to supply `NxFilterInput`s etc as children of the cells.

Reply:
If I understand instead of declaring those properties isFilter, filterPlaceholder, etc..
`            <NxTableCell isFilter
                         filterPlaceholder='Filter'
                         filter={nameFilter}
                         onFilterChange={onFilterNameChange}>
            </NxTableCell>`
The caller (in this case the NxTableFilterExample) should be the one adding the NxFilterInputs as children:
```
          <NxTableRow>
            <NxTableCell>
               <NxFilterInputs .... />
           </NxTableCell>
          </NxTableRow>
```
?

Reply:
> If I understand instead of declaring those properties isFilter, filterPlaceholder, etc.. The caller (in this case the NxTableFilterExample) should be the one adding the NxFilterInputs as children?

Yep, exactly.  The only extra thing that should be needed on the table components is a CSS class or prop which styles the row correctly.

Reply:
How about this?: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/68449f3bbef7a5a73cdf934e28fd69d3114fb2ae